src/status_im2/subs/wallet_2/networks.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.subs.wallet-2.networks
+  (:require [clojure.string :as string]
+            [re-frame.core :as re-frame]
+            [quo2.foundations.resources :as resources]))
+
+(re-frame/reg-sub
+ :wallet-2/filtered-networks-by-mode
+ :<- [:wallet-2/networks]
+ (fn [[networks] [_ test?]]
+   (let [filter-fn (if test? :Test :Prod)]
+     (map filter-fn
+          networks))))
+
+(def default-network-details
+  {:chain-id     1
+   :source       (resources/networks :ethereum)
+   :short-name   "eth"
+   :network-name :ethereum})
+
+(re-frame/reg-sub
+ :wallet-2/network-details
+ :<- [:wallet-2/filtered-networks-by-mode false]
+ (fn [networks]
+   (map
+    (fn [{:keys [chainId shortName chainName]}]
```

Comment:
I guess we should rename the keys to use the kebab case before saving it in the app-db (on-success). 

We probably need to have an ns `status-im2.data-store.wallet` which will contain the helper functions (`<-rpc`) which do the renaming of keys and also the default value insertion.